sdk/eventgrid/ci.yml

```diff
@@ -44,3 +44,6 @@ extends:
       - name: azure-eventgrid
```

Comment:
Just confirming, so the same pipeline will be used for track1 and track2 releases?

Reply:
I believe that's how it would work, but I think we wouldn't approve the Track 1 package when we release.

Reply:
The new changes should mean that the pipelines are now separate